static/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -196,6 +206,36 @@ class EventMetas extends Component<Props, State> {
   }
 }
 
+const BrowserCenter = styled('span')`
+  display: flex;
+  align-items: center;
+  gap: ${space(1)};
+`;
+
+const IconContainer = styled('div')`
+  width: 20px;
+  height: 20px;
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+`;
+
+const BrowserDisplay = ({event}: {event: Event}) => {
```

Comment:
Not a big deal, but I generally prefer adding a return type to all functions.
I'll leave it up to you tho, not a strong opinon.